event-details-boundary/Details.test.tsx

```diff
@@ -192,20 +192,14 @@ describe("EventDetailsLoading tests", () => {
   describe("LoadEventDetails tests", () => {
     fakeTimers()
 
-    it("should return canceled when a 204 occurs", async () => {
```

Comment:
Currently backend returns event details regardless of when the event ended, so we should revisit this in the future